src/treelearner/leaf_splits.hpp

```diff
@@ -53,6 +53,24 @@ class LeafSplits {
     weight_ = weight;
   }
 
+  /*!
+  * \brief Init split on current leaf on partial data. 
+  * \param leaf Index of current leaf
+  * \param data_partition current data partition
+  * \param sum_gradients
+  * \param sum_hessians
+  * \param sum_gradients_and_hessians
```

Comment:
```suggestion
  * \param sum_gradients_and_hessians
  * \param weight
```

Looks like this one was missed